src/main.rs

```diff
@@ -132,7 +111,7 @@ async fn main() {
         .parse::<u64>()
         .expect("UI_PORT format");
 
-    create_module_certificate().await;
+    let _ = certificate::create_module_certificate(&cert_path!(), &key_path!()).await;
```

Comment:
what should we do in error case @JanZachmann

Reply:
I would `panic!`

Reply:
done